portfolio/src/main/webapp/gallery.html

```diff
@@ -4,19 +4,29 @@
     <meta charset="UTF-8">
     <title>Saakshi's Picture Gallery</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
 
   <body>
+    <navbar>
+            <ul>
+                <li><a href="index.html#about-me">About me</a></li>       
```

Comment:
I've tried the navbar in GitHub previews and it stays on the gallery page. I'm not sure if this is an issue of the GitHub priview though. Can you test it on your server please?

Reply:
I tested the navbar again (on the cloud console preview) and it takes me to all the links as supposed to. I'll send you the link after deploying it, could you please check once again then? Thanks.